src/client/app/controllers/join.controller.js

```diff
@@ -0,0 +1,32 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('poker')
+    .controller('JoinController', JoinController);
+
+  /* @ngInject */
+  JoinController.$inject = ['toastr', 'PokerService'];
+
+  function JoinController(toastr, PokerService) {
+    var vm = this;
+    vm.getJoinInfo = getJoinInfo;
+
+    function getJoinInfo(joinInfo) {
+
+      var success = function(response) {
+        console.log([response.user_guid, reponse.game_guid]);
+        vm.user = response.player;
+        vm.game = response.game_guid;
```

Comment:
Need to put this in memory or local storage so we can access it all over the place.  Guessing local storage would be ok?  Should also be `player_guid`.  Should put your name in localStorage as well.